### DISCUSSION TITLE
大佬请问mpv.net带的config编辑器是如何汉化的呀？

### DISCUSSION THREAD

**Discussion by htyxyt** [hooke007/mpv_PlayKit#488] [libmpv]
我看你是做了汉化，如果我想自己从官方的做翻译，该修改哪个文件呀？

**Comment by hooke007** [comment]
mpvnetcm不是汉化，重做了编辑器。现在的mpvnet自带多语言框架

**Reply by htyxyt** [reply]
> mpvnetcm不是汉化，重做了编辑器。现在的mpvnet自带多语言框架

谢谢，明白了
